### DISCUSSION TITLE
Welcome to OpenCalc Discussions!

### DISCUSSION THREAD

**Discussion by clementwzk** [clementwzk/OpenCalc#91] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by OmlineEditor** [comment]
please make it possible to change the position where the arrow is shown. This will reduce the size of the keyboard so that it is convenient to work with one hand on a large screen. And secondly, increase the size of the visible area so that it is convenient to look at long expressions on several lines, and not on one short line.

![3](https://user-images.githubusercontent.com/13348933/202910992-72506137-fe96-4851-85d2-aef5f51229da.png)
